pkg/loop/internal/relayer.go

```diff
@@ -370,6 +370,7 @@ func (r *relayerServer) newMedianProvider(ctx context.Context, relayArgs types.R
 		pb.RegisterContractTransmitterServer(s, &contractTransmitterServer{impl: provider.ContractTransmitter()})
 		pb.RegisterReportCodecServer(s, &reportCodecServer{impl: provider.ReportCodec()})
 		pb.RegisterMedianContractServer(s, &medianContractServer{impl: provider.MedianContract()})
+		//pb.RegisterChainReaderServer(s, &chainReaderServer{impl: provider.ChainReader()})
```

Comment:
This seems important?
How were tests passing? Shouldn't a static test have failed?

Reply:
These are all commented out as part of the same commit, whose purpose is to disable ChainReader temporarily while we merge the evm,starknet,solana, & cosmos PR's so that it doesn't break on the develop branch while we're merging them:
https://github.com/smartcontractkit/chainlink-relay/commit/cd7d4e437e026ff154caea82f2eab18f2f15e60a

The commit immediately adjacent to it removes the refs which were pointing to the branch on each of those repos corresponding to this PR.  These two go together, so everything in CI passes either with them both in or with them both out.  So the plan is, as soon as the other 4 are merged then we just revert this one but not the other and they should pass again.

Reply:
This complexity has never been necessary before. We need to first review everything in a stable state. After that we can simply merge in order and update the deps at each level as we go. No follow up or workarounds are necessary. The only catch is that some repos have CI which is too strict. These can be ignored as long as the core PR at the top builds. Authorized users can bypass the required CI steps after confirming that the core side builds.

Reply:
When you were asking about the `MergeValueFields` and `SplitValueFields` I said there was no entanglement, but there is here: there is no order because this depends on the other 4 being in first and the other 4 all depend on this being in first.  That's why we have to do this one in two stages... one before and one after.  If you prefer... instead of uncommenting the things which have to wait till after I can just delete those lines during the merge process... but I don't see any better way of getting these 5 PR's.